config/routes.rb

```diff
@@ -1,12 +1,16 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
-    get 'home/index'
-    resources :posts
-    root 'static_pages#home'
-    get 'static_pages/help'
-    get 'static_pages/contact'
+
+    root :to => 'static_pages#home', as: 'home'
+    get "/help", to: "static_pages#help", as: "help"
+    get "/contact", to: "static_pages#contact", as: "contact"
+
     get "/signup", to: "users#new"
     post "/signup", to: "users#create"
-    resources :users, only: %i(new create show)
+    resources :users, only: %i(show new create destroy)
```

Comment:
why you have a route for destroy action?

Reply:
i just follow coding in the video. my other friends also include destroy. should i delete it?